lightning/src/offers/invoice_request.rs

```diff
@@ -518,19 +527,35 @@ where
 
 impl UnsignedInvoiceRequest {
 	fn new(offer: &Offer, contents: InvoiceRequestContents) -> Self {
+		let mut bytes = Vec::new();
```

Comment:
Should this also use `with_capacity`, and below?

Reply:
Latest push uses `with_capacity` for `UnsignedStaticInvoice` as well.